release_images.yml

```diff
@@ -50,3 +50,28 @@ release_images:
       os_version: "ubuntu20.04"
       example: False
       disable_sm_tag: True
```

Comment:
remove all the entries above since these are added in the .release_images_template.yml file